full/Dockerfile

```diff
@@ -137,34 +137,36 @@ ENV GOPATH=/workspace/go \
 ### Java ###
 ## Place '.gradle' and 'm2-repository' in /workspace because (1) that's a fast volume, (2) it survives workspace-restarts and (3) it can be warmed-up by pre-builds.
 RUN curl -s "https://get.sdkman.io" | bash \
- && bash -c ". /home/gitpod/.sdkman/bin/sdkman-init.sh \
-             && sdk install java 8.0.202-zulufx \
-             && sdk install java 11.0.2-zulufx \
-             && sdk default java 8.0.202-zulufx \
-             && sdk install gradle \
-             && sdk install maven \
-             && mkdir /home/gitpod/.m2 \
-             && printf '<settings>\n  <localRepository>/workspace/m2-repository/</localRepository>\n</settings>\n' > /home/gitpod/.m2/settings.xml"
+    && bash -c ". /home/gitpod/.sdkman/bin/sdkman-init.sh \
+    && sdk install java 8.0.202-zulufx \
```

Comment:
This indentation change makes a bit confusing what's part of the `bash -c ""` and what not.

Reply:
indeed. was unintended. but the build is through and will first test it. IF everything works I'll remove the indentation changes and merge. Ok?